.github/workflows/build-rails-base.yml

```diff
@@ -27,8 +30,8 @@ jobs:
     - name: prepare
       run: |-
         apk add --no-cache python3 py3-pip
-        pip3 install --upgrade pip
-        pip3 install awscli
+        pip3 install --break-system-packages --upgrade pip
+        pip3 install --break-system-packages awscli
```

Comment:
Did you face any problems in this step for the 3.2.3 image?

Reply:
Yes, see [`ccf5de2` (#21)](https://github.com/degica/dockerfiles/pull/21/commits/ccf5de27d5fd305f6cfbf4bb8ac0990c3245cf1a).

Reply:
Ah, thanks for your explanation in detail!